app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -71,6 +76,10 @@ class TabsTrayFragment : AppCompatDialogFragment(), TabsTrayInteractor {
             removeUseCases
         )
 
+        consumeFrom(requireComponents.core.store) {
+            updateState()
```

Comment:
nit: pass the specific thing we want to consume to the method instead of making this a generic method name. This would avoid a catch-all method to throw everything else in it.

Alternatively, observe changes to just the `normalTabs` instead of using `consumeFrom` which observes all state changes.

Reply:
Agreed